libpod/container_api.go

```diff
@@ -542,12 +545,13 @@ func (c *Container) Inspect(size bool) (*inspect.ContainerInspectData, error) {
 	if err != nil {
 		return nil, errors.Wrapf(err, "error getting graph driver info %q", c.ID())
 	}
-
+	defer c.newContainerEvent(events.Inspect)
```

Comment:
Does this count as an event? We're not modifying the container at all

Reply:
No I don't think we should include this.  Reading info is not an event.  As far as I am concerned.